.editorconfig

```diff
@@ -0,0 +1,15 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[Makefile]
+indent_style = tab
```

Comment:
There is no Makefile, but it will be a very good idea to use it.
So you can change your Readme by,

* Run coke
```sh
make coke
```

* Run Tests
```sh
make test
```